department.rb

```diff
@@ -0,0 +1,39 @@
+require './employee'
+
+class Department
+
+  attr_reader :name, :employees
+
+  def initialize(name)
+    @name = name
+    @employees = []
+    @top_performers = []
+  end
+
+  def add_employee(employee)
+    @employees << employee
+  end
+
+  def total_salaries
+    # total = 0
```

Comment:
When you commit code, it's good to have cleaned out all commented lines like this first.  If you've used git properly, these sorts of things will be present in your git history if you ever need to pull them out.